packages/menhir/menhir.20170509/url

```diff
@@ -0,0 +1,2 @@
+archive: "http://gallium.inria.fr/~fpottier/menhir/menhir-20170509.tar.gz"
```

Comment:
This URL is currently returning 404.  Is it just a temporary problem, @fpottier?